.github/workflows/ksud.yml

```diff
@@ -29,11 +29,7 @@ jobs:
         restore-keys: ${{ runner.os }}-cargo-
 
     - name: Build ksud
-      uses: actions-rs/cargo@v1
-      with:
-        use-cross: true
-        command: build
-        args: --target ${{ inputs.target }} --release --manifest-path ./userspace/ksud/Cargo.toml
+      run: cross build --target ${{ inputs.target }} --release --manifest-path ./userspace/ksud/Cargo.toml
```

Comment:
when the `cross` is installed?

Reply:
seems built-in in action's ubuntu image

Reply:
According to https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md, it is included in the action image.